### ISSUE TITLE
Should grids use flexbox and inline-block fallback?

### ISSUE THREAD

**Issue by jtuds** [cartridge/cartridge-sass#154]
Flexbox grids will allow for better grid layouts and an inline-block fallback would mean the layouts still work on older browsers.

**Comment by mmacartney10** [comment]
Yeah I agree with this, we need to ensure that people know the webpage won't look the same on each browser with this.

**Comment by jtuds** [comment]
Maybe even future proof it with grids and go for a  grid > flexbox > inline-block fallback

**Comment by mmacartney10** [comment]
I wouldn't add grid just yet, flexbox is good because the browser support is really good. But grids have no chance of being used anywhere currently and will just add unnecessary styles.

**Comment by tawashley** [comment]
Grids support should be a separate issue also I reckon 👍
